internal/controller/controllers/infraenv_controller.go

```diff
@@ -525,6 +525,8 @@ func (r *InfraEnvReconciler) setSignedBootArtifactURLs(infraEnv *aiv1beta1.Infra
 		return err
 	}
 	baseURL, err := url.Parse(r.ServiceBaseURL)
+	// ipxeScript
```

Comment:
I think this should either be removed or expanded to make it a bit more informative.